bangc-ops/mlu_op.h

```diff
@@ -283,14 +288,14 @@ mluOpDiv(mluOpHandle_t handle, const mluOpComputationPreference_t prefer,
  *  - The group_size should be equal to pooled_height.
  *  - The pooled_height should be equal to pooled_width.
  *  - The channels should be equal to pooled_height * pooled_width * output_dim.
- *  - The dim of input_data should be equal to 4.
- *  - The dim of input_rois should be equal to 2.
- *  - The dim of output should be equal to 4.
- *  - The dim of mapping_channel should be equal to 4.
+ *  - The dimension of input should be equal to 4.
```

Comment:
```suggestion
 *  - The dimension of \b input should be equal to 4.
```
后面的注释也排查一下，入参都加上\b渲染